src/LearnerData/SFA.DAS.LearnerData/Validators/LearnerDataRequestValidator.cs

```diff
@@ -3,57 +3,86 @@
 
-        RuleFor(model => model.StandardCode).GreaterThanOrEqualTo(0)
-            .WithMessage(model => $"Learner data contains a negative StandardCode {model.StandardCode}");
+        RuleFor(model => model.Delivery.OnProgramme.StandardCode).GreaterThanOrEqualTo(0)
+            .WithMessage(model => $"Learner data contains a negative StandardCode {model.Delivery.OnProgramme.StandardCode}");
 
-        RuleFor(model => model.ConsumerReference)
-            .MaximumLength(100)
-            .WithMessage("ConsumerReference cannot be more then 100 characters long");
+       RuleFor(model => model.ConsumerReference)
+           .MaximumLength(100)
+           .WithMessage("ConsumerReference cannot be more then 100 characters long");
 
-        RuleFor(model => model.FirstName)
+       RuleFor(model => model.Learner.Firstname)
             .NotEmpty()
-            .WithMessage("FirstName is required")
+            .WithMessage("Firstname is required")
             .MaximumLength(100)
-            .WithMessage("FirstName cannot be more then 100 characters long");
+            .WithMessage("Firstname cannot be more then 100 characters long");
 
-        RuleFor(model => model.LastName)
+        RuleFor(model => model.Learner.Lastname)
             .NotEmpty()
-            .WithMessage("LastName is required")
+            .WithMessage("Lastname is required")
             .MaximumLength(100)
-            .WithMessage("LastName cannot be more then 100 characters long");
+            .WithMessage("Lastname cannot be more then 100 characters long");
+
+        RuleFor(model => model.Learner.Dob)
+            .NotNull()
+            .WithMessage("Dob is required");
 
-        RuleFor(model => model.LearnerEmail)
+        RuleFor(model => model.Learner.Email)
             .MaximumLength(200)
             .WithMessage("Email cannot be more then 200 characters long")
-            .EmailAddress().When(model => model.LearnerEmail != null);
+            .EmailAddress().When(model => model.Learner.Email != null);
 
-        RuleFor(model => model.AgreementId)
+        RuleFor(model => model.Delivery.OnProgramme.AgreementId)
             .MaximumLength(20)
-            .WithMessage("AgreementId cannot be more then 20 characters long");
+            .WithMessage("OnProgramme AgreementId cannot be more then 20 characters long");
+
+        RuleFor(model => model.Delivery.OnProgramme.StartDate)
+            .NotNull()
+            .WithMessage("OnProgramme StartDate is required");
+
+        RuleFor(model => model.Delivery.OnProgramme.ExpectedEndDate)
+            .NotNull()
+            .WithMessage("OnProgramme ExpectedEndDate is required");
+
+        RuleFor(model => model.Delivery.OnProgramme.IsFlexiJob)
+            .NotNull()
+            .WithMessage("OnProgramme IsFlexiJob is required");
     }
+
+    private bool ValidateUln(string uln)
```

Comment:
There is checksum validation you can do on ULN, not sure if that is necessary here